paper/forestecology.Rmd

```diff
@@ -213,10 +209,12 @@ growth_scbi <- growth_scbi %>%
   add_buffer_variable(size = comp_dist, region = study_region_scbi)
 ```
 
-The second element of spatial information are blocks corresponding to folds of a spatial cross-validation algorithm used to estimate out-of-sample model error. Conventional cross-validation algorithms assign observations to folds by randomly resampling individual observations. However, an assumption that the observations are independent underlies any cross-validation algorithm. In the case of forest census data, observations exhibit spatial autocorrelation. We therefore incorporate this spatial dependence into the cross-validation algorithm by randomly resampling spatial blocks of trees @roberts_cross-validation_2017 @pohjankukka_estimating_2017.
+The second element of spatial information are blocks corresponding to folds of a spatial cross-validation algorithm used to estimate out-of-sample model error. Conventional cross-validation algorithms assign observations to folds by randomly resampling individual observations. However, an assumption that the observations are independent underlies any cross-validation algorithm. In the case of forest census data, observations exhibit spatial autocorrelation. We therefore incorporate this spatial dependence into the cross-validation algorithm by randomly resampling spatial blocks of trees @roberts_cross-validation_2017 @pohjankukka_estimating_2017. (DAVE NOTE: I don't want to change somethign I don't understand, but "randomly resampling" seems wrong to me. I don't see anything random about our block structure. Seems to me like "incorpoate this spatial dependence into the cross-validation algorithm with our spatial blocks of trees.")
```

Comment:
you're correct. in regular CV, the observations get assigned randomly to folds. Here, they are assigned depending on their location